src/WebApi/Infrastructure/Middleware/BasicAuthMiddleware.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using System.Net.Http.Headers;
+
+namespace Sindibad.SAD.WebTemplate.WebApi.Infrastructure.Middleware
+{
+    public class BasicAuthMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private const string Realm = "SindibadAPI";
+
+        public BasicAuthMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            if (!context.Request.Path.Value.Contains("api"))
```

Comment:
I don't think checking the request path with `Contains` is safe. There can easily be an edge case to this. Also, why are we allowing none "api" paths here to go through authorization?